src/main/cli/cli.c

```diff
@@ -1033,12 +1034,12 @@ static void cliAux(char *cmdline)
             analyzeModeActivationConditions();
             cliPrintLinef( "aux %u %u %u %u %u %u %u",
                 i,
-                mac->modeId,
+                findBoxByBoxId(mac->modeId)->permanentId,
```

Comment:
I have a vague memory of us looking into making this change all the way back when the permanent ids for modes were introduced, and then deciding to not do it for a specific reason, but I can not remember what the reason was for the life of me.

One thing that will be problematic with this change is that, for unsuspecting users applying a `diff` from a previous version, this will result in most of their switch functions being changed. We have to mention this in the release notes, but even then many users will probably apply a `diff` without bothering to read the release notes, so I am almost considering something like renaming the command, in order to make the restore fail, and give the user a prompt to manually fix things.

Reply:
@ledvinap: I think you were involved in this discussion - do you remember?

Reply:
I don't think it's a good idea at all to break the restoration of switch settings.  one of the switch setting is of course ARM, using a different identifier for ARM could well be a safety issue.

not in favor of changing the cli command name.

clear instructions on how to migrate old switch settings must be provided.

don't we have strings for the name of each setting in the firmware?  if so we could make the command accept/print strings instead of numerical identifiers which would likely be a better and more robust solution.

Reply:
@hydra See my comments https://github.com/betaflight/betaflight/pull/7595#issuecomment-464456759

There is no problem with previous settings. It's just a minor bug on the output after the actual command. The `diff` output that would be used to migrate settings is correct.